### PR TITLE
Write routing configuration using scr directly

### PR DESCRIPTION
**Pull request by mchf**
TODO: unit tests

### CODE REVIEW

**Comment by mchf** [comment]
Fixed remote testsuite failures
Some cleanup and squashed commits a bit

File: src/lib/y2network/sysconfig_routes.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      )
+    end
+
+    # Registers SCR agent which is used for accessing particular ifroute-device
+    # file
+    #
+    # @param device [String] full path to a file in routes format (e.g. /etc/sysconfig/network/ifroute-eth0)
+    # @return [Path] SCR path of the agent
+    # @raise  [RuntimeError] if it fails
+    def register_ifroute_agent_for_path(path)
```
Comment (jreidinger) [inline src/lib/y2network/sysconfig_routes.rb]: hmm, this looks a bit duplicite to @imobachgs pull request, where I already comment this part of code.

Comment (mchf) [reply src/lib/y2network/sysconfig_routes.rb]: in fact it is copy of what already was in y2-network and copied into new routing here: https://github.com/yast/yast-network/pull/750

Comment (imobachgs) [reply src/lib/y2network/sysconfig_routes.rb]: @jreidinger We discussed yesterday that we would merge this PR (which contains the tested logic to write changes) and I will rebase my changes on top to move this logic into `Y2Network::SysconfigRoutesFile` class.

Comment (jreidinger) [reply src/lib/y2network/sysconfig_routes.rb]: ok, it is fine then.

**Pull request merged**